### ISSUE TITLE
Mac OSX package not installing

### ISSUE THREAD

**Issue by barambani** [dotnet/cli#1619]
The Mac OS X package downloaded from here

https://github.com/dotnet/cli

looks not able to install CLI. The installation procedure succeeds but the terminal cannot find the dotnet command.
Is there any way to verify that the installation was actually completed ? There's nothing in my /usr/local/bin  

Thanks

**Comment by blackdwarf** [comment]
@barambani which shell are you using? Can you check if there is something in `/usr/local/share/dotnet`?

**Comment by barambani** [comment]
@blackdwarf thanks for the help. Both the folders contain files (see attachments). It looks like the installation is ok then.
I use zsh as shell

![screen shot 2016-02-27 at 20 22 13](https://cloud.githubusercontent.com/assets/201339/13375141/fb3db36e-dd8f-11e5-8f8c-10450c01e627.png)
![screen shot 2016-02-27 at 20 21 27](https://cloud.githubusercontent.com/assets/201339/13375143/028f1590-dd90-11e5-87b1-9e3b4aef21e3.png)

**Comment by blackdwarf** [comment]
@barambani yes, the problem is in the way path is getting built on OS X. If you start bash, you will have access. There is another issue on this at #1567. I will close this as a duplicate if that is ok with you. 

In the meantime, you can get unblocked by simply adding `/usr/local/share/dotnet/bin/` to $PATH.

**Comment by barambani** [comment]
Great, thanks a lot
